### ISSUE TITLE
Improved url support + 3rd party support

### ISSUE THREAD

**Issue by SfietKonstantin** [sailfishos/sailfish-browser#34]
Be able to open an application when clicking on an url. Right now, http(s):// mailto:// and tel:// are supported, but maybe we should have extended support, like clicking on a http://twitter.com/<something> opens Android Twitter / Tweetian.

This need 3rd party support to be able to handle this too.

**Comment by jklingen** [comment]
I agree, it would be great if the user was able to send the URL of the current page to another app, e.g. to a password generator like [SuperSailPass](https://github.com/jklingen/harbour-supersailpass) :smirk:

Other valid use cases could be:
* send to barcode generator
* send to url shortener
* share with [whatever]
* ...

But there's no equivalent to Android's `Intent.ACTION_VIEW`, right?
